content/kots-cli/admin-console/generate-manifests.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2021-07-29
+linktitle: "generate-manifests"
+title: kots admin-console generate-manifests
+weight: 90110
```

Comment:
there's already a command with this weight. i would suggest usin `90130`

Reply:
done, good catch!